### PR TITLE
Updated Readme with Examples

### PR DESCRIPTION
**Pull request by Burgi0101**
Added examples for Terminal & NPM Script usage including Windows support!
@mikeal Please check it and tell me if there is anything that I should change!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,28 @@ OPTIONS:
...
+    
+As the command has to be written in quotation marks Windows users may need to use double quotes rather than single quotes
+
+    watch "npm run test -s" directory
+    
+Note: Because Windows users may need to use double quotes rather than single quotes they need to escape the them in npm scripts
+    
+    "scripts": {
+        "watch:osx": "watch 'npm run test -s' source"
+        "watch:windows": "watch "npm run test -s" source",
```
Comment (levithomason) [inline README.md]: Whoops, escaping looks like it was missed here.

**Review by levithomason — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Burgi0101) [inline README.md]: I think escaped quotationmarks are just not shown excaped on github!

Comment (levithomason) [inline README.md]: Looks like they are missing in the raw as well:

image

https://raw.githubusercontent.com/mikeal/watch/fad8de91b5aedea105f336acd4171a10beb805a9/README.md

Comment (levithomason) [inline README.md]: Perhaps they got removed on commit somehow?  Do you see them locally?

Comment (Burgi0101) [inline README.md]: I have to check this, I am currently on my phone! Will try to commit again with escaped double quotes and then we'll see whats the issue!
Thx for pointing this out!

Comment (levithomason) [inline README.md]: No worries my first comment was on my phone and I also hesitated at first :P

Comment (Burgi0101) [inline README.md]: yeah it's just not shown in the GUI here!

Comment (levithomason) [inline README.md]: well, I cannot pull the branch as GitHub doesn't know where the repo is:

image


But, if I down the raw file, I get this:

image

I've never seen the unknown repo thing before, but, perhaps it is related.  What is clear is that the changes in this PR do not include escaped quotes.

**Comment by levithomason** [comment]
Just merged this in #135.

**Pull request closed without merging**